Dockerfile

```diff
@@ -12,6 +12,8 @@ WORKDIR /app
 EXPOSE 8080
 ENV PORT=8080 MIX_ENV=prod
 
+ENTRYPOINT ["/usr/bin/dumb-init", "./load_secrets_and_run.sh"]
```

Comment:
i think this has to be right above [CMD](https://github.com/artsy/aprd/blob/c6abf0508d35bac1877ddaafff6afbdd618485af/Dockerfile#L32).

Reply:
Oh, i didn't know that. Updated.